x/metadata/keeper/objectstore.go

```diff
@@ -189,14 +141,11 @@ func (k Keeper) ModifyRecord(ctx sdk.Context, ownerAddr sdk.AccAddress, uri stri
 	return nil
 }
 
-// ImportLocatorRecord binds a name to an address. An error is returned if no account exists for the address.
+// ImportLocatorRecord binds a name to an address in the kvstore.
+// Different from SetOSLocatorRecord in there is less validation.
```

Comment:
As noted in the other comment, the use of the auth module to check if an account exists when the normal msg call path has already required it to be the signer and pay the gas fees is an invalid constraint that leads to a duplicate method like this one for setting locators during genesis or similar flows where the dependency/order of account creation can not be determined reliably ahead of time.